### PR TITLE
Edit nameserver record

### LINKED ISSUE #1048: Mockup deemphasize cancel buttons
The blue attracts more attention, open to potential solutions.

### LINKED ISSUE #1139: Implement add / edit Nameserver record

### PR DESCRIPTION
**Pull request by eatonphil**
Closes #1048
Closes #1139

screen shot 2017-02-15 at 9 24 27 am
screen shot 2017-02-15 at 11 59 40 am

### CODE REVIEW

File: scss/components/buttons.scss [hunk 1]
```diff
@@ -140,6 +140,23 @@
...
+
+    &.btn-secondary {
+        border: $border;
+        border-color: #eee;
+        color: #777;
+        background: $white;
+
+        &:not([disabled]):hover {
+            border: $border;
+            color: #4b4b4b;
```
Comment (mparke) [inline scss/components/buttons.scss]: color var?

File: scss/components/buttons.scss [hunk 2]
```diff
@@ -140,6 +140,23 @@
         margin-left: 5px;
         padding: 8px;
     }
+
+    &.btn-secondary {
+        border: $border;
+        border-color: #eee;
```
Comment (mparke) [inline scss/components/buttons.scss]: more color vars, do these exist?

File: src/dnsmanager/components/EditNSRecord.js [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+            id="ttl"
+            name="ttl"
+            value={ttl}
+            defaultSeconds={defaultTTL}
+            onChange={this.onChange}
+          />
+        </ModalFormGroup>
+        <div className="Modal-footer">
+          <CancelButton onClick={close} />
+          <SubmitButton disabled={saving}>{this.state.id ? 'Save' : 'Add NS Record'}</SubmitButton>
```
Comment (mparke) [inline src/dnsmanager/components/EditNSRecord.js]: Screenshot shows header text `Edit ...` and `Add NS Record` for the action

File: scss/components/buttons.scss (hunk 1 above)
Comment (eatonphil) [inline scss/components/buttons.scss]: Oops, forgot to change this out.

File: scss/components/buttons.scss (hunk 2 above)
Comment (eatonphil) [inline scss/components/buttons.scss]: Forgot to switch this out.

File: src/dnsmanager/components/EditNSRecord.js (hunk 3 above)
Comment (eatonphil) [inline src/dnsmanager/components/EditNSRecord.js]: Oh good call.

File: src/dnsmanager/components/SelectDNSSeconds.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  86400: '1 day',
+  172800: '2 days',
+  345600: '4 days',
+  604800: '1 week',
+  1209600: '2 weeks',
+  2419200: '4 weeks',
+};
+
+export function formatDNSSeconds(dnsSeconds, defaultSeconds = 300) {
+  // == is intentional, either could be a string
```
Comment (mparke) [inline src/dnsmanager/components/SelectDNSSeconds.js]: either could be a string?

Comment (eatonphil) [inline src/dnsmanager/components/SelectDNSSeconds.js]: Yeah one or the other is fine to be a string or number type, but they are numbers.

File: src/dnsmanager/components/EditNSRecord.js (hunk 3 above)
Comment (eatonphil) [inline src/dnsmanager/components/EditNSRecord.js]: Screenshot was just wrong, updated.

Comment (eatonphil) [inline src/dnsmanager/components/EditNSRecord.js]: Whoops again, screenshot was not wrong. Pushing fix.

File: src/dnsmanager/components/SelectDNSSeconds.js (hunk 4 above)
Comment (mparke) [inline src/dnsmanager/components/SelectDNSSeconds.js]: I should have been more specific, why not just always work on numbers, and `parseInt` wherever this is returned as a string

File: src/dnsmanager/layouts/ZonePage.js [hunk 5]
```diff
@@ -116,52 +125,94 @@ export class ZonePage extends Component {
       <Card
         title={title}
         id={id}
-        nav={nav || <Button onClick={navOnClick}>Add {titleSingular}</Button>}
+        nav={nav === undefined ? <Button onClick={navOnClick}>Add {titleSingular}</Button> : null}
```
Comment (eatonphil) [inline src/dnsmanager/layouts/ZonePage.js]: Hmm, this does look weird now. I think the last part is supposed to be `nav` not `null`. Will fix.

**Comment by mparke** [comment]
@eatonphil Not specifically related to this pull, but dns pages are broken in master right now for me, we should fix so we can test this before merging. Can you verify?

**Comment by mparke** [comment]
Undefined values in TTL?

screen shot 2017-02-16 at 3 16 48 pm

we should denote seconds (`s` or `seconds`) as the primary value in the TTL

also, this is my domain, but add/edit label denotes subdomain, is confusing:

screen shot 2017-02-16 at 3 18 16 pm

**Pull request merged**